08-rails/code_samples/destroy_controller_tests.rb

```diff
@@ -0,0 +1,29 @@
+describe "destroy" do
+  describe "destroy" do
+    it "can destroy a model" do
+      # Arrange
+      poodr = Book.new title: "Practical Object Oriented Programming in Ruby", author: "Sandi Metz"
+
+      poodr.save
+      id = poodr.id
+
+      # Act
+      expect {
+        delete book_path(id)
+
+        # Assert
+      }.must_change "Book.count", -1
```

Comment:
Should these be `do` `end` blocks or is `expect` weird?

Reply:
Just continuing the pattern we established in regular rails testing.